.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release
 on:
   push:
     tags:
-      - "@kiwicom/[a-z-]+@[0-9]+.[0-9]+.[0-9]+"
+      - "@kiwicom/*@[0-9]+.[0-9]+.[0-9]+"
```

Comment:
this should work better `@kiwicom/*@[0-9]*.[0-9]*.[0-9]*`

Reply:
The thing is that in workflows `*` doesn't refer to preceding characters, like `+` does. The pattern you suggested would match `@kiwicom/orbit-components@0ab.92.0a1c_`.

Reply:
Why do you propose `*` over `+`?

Reply:
well, i tried it [here](https://globster.xyz/) and it did not work, but you are right, + should match preceding chars, so never mind :)

Reply:
You'll tell me tomorrow what exactly didn't work because globster is for matching files, and we're matching the tag name pattern.